rewrite-maven/src/main/java/org/openrewrite/maven/RemoveExclusion.java

```diff
@@ -66,17 +68,19 @@ public Xml visitTag(Xml.Tag tag, ExecutionContext executionContext) {
                 if (isDependencyTag(groupId, artifactId) || isManagedDependencyTag(groupId, artifactId)) {
                     Optional<Xml.Tag> maybeExclusions = tag.getChild("exclusions");
                     if (maybeExclusions.isPresent()) {
-                        return tag.withContent(ListUtils.map(tag.getChildren(), child -> {
-                            if("exclusions".equals(child.getName())) {
-                                Xml.Tag e = child;
+                        return tag.withContent(ListUtils.map((List<Content>) tag.getContent(), child -> {
```

Comment:
just a little comment to keep the ide happy
```suggestion
                        //noinspection unchecked
                        return tag.withContent(ListUtils.map((List<Content>) tag.getContent(), child -> {
```